packages/objects/src/enum/StorageKey.ts

```diff
@@ -40,4 +40,8 @@ export enum EFieldKey {
   DOMAIN_PERMISSIONS = "SD_DomainPermissions",
 }
 
+export enum EExternalFieldKey {
+  UI_STATE = "SD_UIState",
```

Comment:
Not sure if there's a need to break this out. I'll see how you use it before I say anymore though.